card/mod/05_standard/set/all/links.rb

```diff
@@ -31,7 +31,11 @@ def web_link href, opts={}
   # link to a specific card
   def card_link name, opts={}
     text = (opts.delete(:text) || name).to_name.to_show @context_names
-    name = Card[name].cardname if Symbol===name
+    name = case name
+           when Symbol then Card[name].cardname
+           when Card   then name.cardname
+           else             name
+           end
```

Comment:
I'm not sure this is right.  I don't think this method is meant to take a Card, and the symbol case probably should just used Codename and not instantiate a card.  I think this should work for a symbol even if there is no card with that codename.